bin/cli.js

```diff
@@ -131,8 +79,8 @@ yargs
   })
   .fail((err) => {
     // Prints to `stderr` by default
-    return showErrorAndQuit(err);
+    return helpers.showErrorAndQuit(err);
   })
   .argv;
 
-process.on('uncaughtException', showErrorAndQuit);
+process.on('uncaughtException', helpers.showErrorAndQuit);
```

Comment:
I probably wouldn't - it feels clearer and safer to be explicit about error catching, and instead use the event as a backstop to handle anything that gets missed elsewhere. Generally not handling promise rejections is bad behaviour, and we shouldn't actively do it.

It's an interesting point though, because if you remove those catches from inside commands themselves then they provide a quite nice programmatic api to landr, which they don't right now (nobody wants to call a method that will shut down the process if anything goes wrong). If that is worthwhile to us though (I'm not sure - any thoughts?) then instead we should make each commands return its promise, without catching, and then explicitly catch and show helper info in here, rather than not catching at all. Might not be worth it though.